app/controllers/sessions_controller.rb

```diff
@@ -9,7 +9,8 @@ def create
       session[:user_id] = user.id
       redirect_to dashboard_path
     else
-      flash[:error] = "Invalid login credentials"
+      account_link = "#{view_context.link_to("Create new account?", new_user_path)}"
```

Comment:
Line is too long. [84/80]<br>Prefer single-quoted strings inside interpolations.